modules/rosa-oidc-understanding.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * rosa_architecture/rosa-sts-about-iam-resources.adoc
+// * rosa_architecture/rosa_policy_service_definition/rosa-oidc-overview.adoc
+
+:_content-type: CONCEPT
+[id=rosa-oidc-understanding_{context}]
+= Purpose of OpenID Connect Configurations
+
+The OpenID Connect (OIDC) configuration offers a way to set up your OIDC dependencies before you create your cluster. This 
+
+The main change is being able to setup the dependencies prior to cluster spec, which would allow the cluster to move to install faster implying a smaller duration to get to ready. However for classic clusters it takes 40+ minutes so the advantage here isn't that impacful. OIDC Configurations can also be reused by more clusters, but I'm not sure if this is something we'd like to advertised
```

Comment:
Is this something we want to say? 
"but I'm not sure if this is something we'd like to advertised"